src/Converter/Feature2Mesh.js

```diff
@@ -188,13 +168,18 @@ function featureToLine(feature, options) {
 
     let lines;
 
+    // TODO CREATE material for each feature
```

Comment:
By this `TODO`, do you mean that the `var lineMaterial = new THREE.LineBasicMaterial();` on line 148 should later be here instead ?

Reply:
it needs of new PR